### PR TITLE
Add template to Generator

### PR DESCRIPTION
**Pull request by samuel-duhaime**
# Pull request
fix: #414 

## Description
Add template type
Add template to sectionConfigs.ts

### CODE REVIEW

**Review by tahini — CHANGES_REQUESTED** [review]
Could you explain in the commit message or in the README how to add a template? iiuc, we need to put the `template.tsx` file in the survey's corresponding section? And the template should be a React component?

**Comment by samuel-duhaime** [comment]
> Could you explain in the commit message or in the README how to add a template? iiuc, we need to put the `template.tsx` file in the survey's corresponding section? And the template should be a React component?

Yes, with the commit, I did explain it on 'Project Structure' of the README of Generator.

**Review by tahini — APPROVED** [review]

File: packages/evolution-generator/src/types/sectionsTypes.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ export type SectionConfig = {
         fr: string;
         en: string;
     };
+    template?: any;
```
[the remark this answers is not in the document]
Comment (samuel-duhaime) [reply packages/evolution-generator/src/types/sectionsTypes.ts:56]: Non, c'est quelque chose de beaucoup plus compliqué. C'est pour cela que j'ai mis any pour l'instant...

image

[the remark this answers is not in the document]
Comment (samuel-duhaime) [reply packages/evolution-generator/src/types/sectionsTypes.ts:57]: Je viens d'ouvrir un issue pour cela #424 .

**Pull request merged**